utility/enum-io.hpp

```diff
@@ -230,7 +230,8 @@
         is.setstate(std::ios::failbit);                                 \
         return is;                                                      \
     }                                                                   \
-    constexpr int enumerationValuesCount(Type) {                        \
+    UTILITY_POSSIBLY_UNUSED constexpr int enumerationValuesCount(Type)  \
```

Comment:
`[[maybe_unused]]`? We've already switched to c++17 so no need for compat macro.